config-schema/config.json

```diff
@@ -749,14 +749,24 @@
       "$ref": "classpath:/metadata-schema/baseSchema.schema.json#/properties/related_material",
       "propertyType": "Supplied",
       "expectedTDRHeader": false,
```

Comment:
Not sure if we even use this `expectedTDRHeader` but I believe it should be set to `true` for all columns that appear in the UI